src/site/stages/build/drupal/graphql/navigation-fragments/sidebar.nav.graphql.js

```diff
@@ -121,5 +121,8 @@ if (cmsFeatureFlags.FEATURE_ALL_HUB_SIDE_NAVS && hubNavNames !== null) {
     recordsBenefitsHubQuery: ${queryFilter('records-benefits-hub')} {
       ${SIDEBAR_QUERY}
     }
+    decisionReviewsHubQuery: ${queryFilter('decision-reviews-benefits-h')} {
```

Comment:
That string is an ID that I got from this Slack thread, https://dsva.slack.com/archives/C52CL1PKQ/p1585670142294200?thread_ts=1585667623.272700&cid=C52CL1PKQ.